src/components/cardbuilder/cardBuilder.js

```diff
@@ -1854,6 +1854,8 @@ import 'programStyles';
             }
         }
 
+/* eslint-enable indent */
+
 export default {
     getCardsHtml: getCardsHtml,
     getDefaultBackgroundClass: getDefaultBackgroundClass,
```

Comment:
Did you forget to remove indent?
And how does ESLint react to this?

Reply:
It's to re-enable indent after the badly indented section. You disable it at the start of the wrongly indented content, then re-enable it after. It's good practice when doing this.

ESLint will now complain about stray ESLint comments that disable stuff for the file without re-enabling them after, so this removes that error from the lint.